src/main/java/by/exadel/internship/service/impl/FormServiceImpl.java

```diff
@@ -53,17 +54,23 @@ public FormFullDTO process(FormRegisterDTO form, MultipartFile file) {
             return createdForm;
         }
 
-        FormFullDTO createdForm = saveForm(form);
+        FormFullDTO createdForm = saveForm(form, internshipIdStr);
 
         log.info("Success to save form, id: {}", createdForm.getId());
 
         return createdForm;
     }
 
-    private FormFullDTO saveForm(FormRegisterDTO formRegisterDTO) {
+    private FormFullDTO saveForm(FormRegisterDTO formRegisterDTO, String internshipIdStr) {
+
+        UUID internshipId = UUID.fromString(internshipIdStr);
 
         Form form = mapper.toFormEntity(formRegisterDTO);
+
+        log.info("Save form, id: {} internshipId {} and status {}", form.getId(), internshipIdStr, FormStatus.REGISTERED);
+
         form.setFormStatus(FormStatus.REGISTERED);
+        form.setInternshipId(internshipId);
```

Comment:
It's UI responsibility to set `internshipId`

Reply:
done